packages/manager/apps/pci-object-storage/src/utils/getStatusColor.ts

```diff
@@ -0,0 +1,15 @@
+export type TStatusColor = 'success' | 'critical' | 'warning' | 'information';
+
+type TGetStatusColorFn<T extends string> = (
+  status: T | undefined,
```

Comment:
in which case status could be undefined?
and what's the point having a generic type T ?